tools/tertiary-analysis/sc3/sc3-sc3-calc-biology.xml

```diff
@@ -1,17 +1,18 @@
-<tool id="sc3-sc3-calc-biology" name="SC3 calculate biology" version="0.1.0">
+<?xml version="1.0" encoding="utf-8"?>
+<tool id="sc3-sc3-calc-biology" name="SC3 calculate biology" version="@TOOL_VERSION@+galaxy0">
     <description>SC3 calculates biologically relevant clusters</description>
     <macros>
-        <import>macros.xml</import>
+        <import>sc3-macros.xml</import>
     </macros>
     <expand macro="requirements" />
     <command detect_errors="exit_code"><![CDATA[
         sc3-sc3-calc-biology.R -i "$input1" -o "$output1"
     ]]></command>
     <inputs>
-        <param type="data" name="input1" format="rds" />
+        <param type="data" name="input1" format="rds" label="File name in which a SC3 'SingleCellExperiment' object has been stored after kmeans clustering"/>
```

Comment:
Please put the label text in the `help` field (which is supposed to for what you have currently in label). In label you should write something like SC3 R serialized object. Format should be rdata and not RDS. I don't think Galaxy actually works with rds, how did you make that work?

Reply:
Please do the same in other inputs and other wrappers.